prov/efa/src/efa_av.c

```diff
@@ -1075,3 +1075,102 @@ int efa_av_open(struct fid_domain *domain_fid, struct fi_av_attr *attr,
 	free(av);
 	return ret;
 }
+
+#ifdef EFA_UNIT_TEST
```

Comment:
As I mentioned before, define EFA_UNIT_TEST to 0 in the else branch and use `if` here instead of `ifdef`.

Reply:
done